src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -110,6 +111,48 @@ static uint16_t               eq_count_max;
 static uint16_t               eq_count;
 static uint16_t               eq_idx;
 
+/* the number of EQs for aio contexts. (aio_eq_count + eq_count) <= eq_count_max */
+static uint16_t               aio_eq_count;
```

Comment:
the name also is confusing here, so would suggest to rename to:
aio_eqs_count_g

Reply:
I just realized that "aio_eq_count" is renamed as "aio_eq_count_g" now. Do I need to change them to "aio_eq**s**_count_g"? Thank you!